core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -124,18 +124,93 @@ describe('Exploration translation', function() {
     explorationEditorMainTab.setInteraction('Continue');
     responseEditor = explorationEditorMainTab.getResponseEditor('default');
     responseEditor.setDestination('final card', true, null);
-    // Setup a terminating state.
     explorationEditorMainTab.moveToState('final card');
     explorationEditorMainTab.setInteraction('EndExploration');
     explorationEditorMainTab.moveToState('first');
     explorationEditorPage.saveChanges();
-
     explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
     explorationEditorTranslationTab.expectNumericalStatusToMatch(
       '(0/8)');
+
+    explorationEditorTranslationTab.openUploadAudioModal();
+    var audioToUpload = '../data/cafe.mp3';
+    var audioAbsolutePath = path.resolve(__dirname, audioToUpload);
+    explorationEditorTranslationTab.audioElem().sendKeys(audioAbsolutePath);
+    explorationEditorTranslationTab.saveUploadedAudio(true);
+    explorationEditorTranslationTab.expectNumericalStatusToMatch(
+      '(1/8)');
+    explorationEditorTranslationTab.openUploadAudioModal();
+
+    var audioToUpload = '../data/img.png';
+    var audioAbsolutePath = path.resolve(__dirname, audioToUpload);
+    explorationEditorTranslationTab.audioElem().sendKeys(audioAbsolutePath);
+    expect(explorationEditorTranslationTab.errorMessage === undefined).
```

Comment:
Can we validate the error message instead of just checking that an error message exists?

Reply:
Done